core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -58,11 +60,11 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
   // Submit tasks only after (registered resources / total expected resources)
   // is equal to at least this value, that is double between 0 and 1.
   private val _minRegisteredRatio =
-    math.min(1, conf.getDouble("spark.scheduler.minRegisteredResourcesRatio", 0))
+    math.min(1, conf.get(SCHEDULER_MIN_REGISTERED_RESOURCES_RATIO).getOrElse(0.0))
```

Comment:
Can this prop just have a default value of 0 where it's declared?

Reply:
We cannot have a defalut value `0` where it is declared due to the interesting code at `KubernetesClusterSchedulerBackend.scala`.  
We need to know `SCHEDULER_MIN_REGISTERED_RESOURCES_RATIO` is declared or not.